src/Workspaces/Core/Portable/Simplification/SimplificationHelpers.cs

```diff
@@ -102,15 +102,15 @@ internal static bool IsValidSymbolInfo(ISymbol symbol)
         }
 
         internal static bool ShouldSimplifyMemberAccessExpression(
-            SemanticModel semanticModel, SyntaxNode expression, OptionSet optionSet)
+            SemanticModel semanticModel, OptionSet optionSet, ISymbol symbol)
         {
-            var symbol = GetOriginalSymbolInfo(semanticModel, expression);
-            if (symbol == null ||
-                (!symbol.IsStatic &&
-                 (symbol.IsKind(SymbolKind.Field) && optionSet.GetOption(CodeStyleOptions.QualifyFieldAccess, semanticModel.Language).Value ||
-                 (symbol.IsKind(SymbolKind.Property) && optionSet.GetOption(CodeStyleOptions.QualifyPropertyAccess, semanticModel.Language).Value) ||
-                 (symbol.IsKind(SymbolKind.Method) && optionSet.GetOption(CodeStyleOptions.QualifyMethodAccess, semanticModel.Language).Value) ||
-                 (symbol.IsKind(SymbolKind.Event) && optionSet.GetOption(CodeStyleOptions.QualifyEventAccess, semanticModel.Language).Value))))
+            if (symbol.IsStatic)
+                return false;
+
+            if ((symbol.IsKind(SymbolKind.Field) && optionSet.GetOption(CodeStyleOptions.QualifyFieldAccess, semanticModel.Language).Value ||
```

Comment:
💭 It's strange to see the outermost parenthesized expression going around the entire condition, as opposed to just surrounding the first `&&` expression.

Reply:
ick yes.  nice catch.